### ISSUE TITLE
[Project Enhancement]: Brain Stroke Prediction using Neural Networks

### ISSUE THREAD

**Issue by Srijansarkar17** [abhisheks008/ML-Crate#605]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Brain Stroke Prediction using Neural Networks** :
:red_circle: **To improve the accuracy of prediction of brain stroke using neural networks** :
:red_circle: **https://www.kaggle.com/datasets/fedesoriano/stroke-prediction-dataset** :
:red_circle: **I will use neural networks for this approach so I will approach this with various hidden layers and different types of activation functions so that the accuracy can be improved.** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Srijan Sarkar
- GitHub Profile Link : https://github.com/Srijansarkar17
- Participant ID (If not, then put NA) : NA
- Approach for this Project : I will use neural networks for this approach so I will approach this with various hidden layers and different types of activation functions so that the accuracy can be improved.
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)   SSOC

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Srijansarkar17** [comment]
Hey,
Can You Please Assign this issue under SSOC. 2024 Season 3
Srijan Sarkar
Github:- https://github.com/Srijansarkar17
Participation ID:- NA
I am good at making projects using neural networks. So can you please assign this task to me?
Thanks.
Participation Role:- SSOC Season 3

**Comment by Armaan457** [comment]
Armaan Jagirdar
Github: https://github.com/Armaan457
Participating under SSOC'24
I have the working knowledge of Neural Networks and CNNs and would really like to work on this. I'll be implementing this using TensorFlow trying different activation functions and layers. Can I be assigned this?
Thank you!

**Comment by Armaan457** [comment]
> > Armaan Jagirdar Github: https://github.com/Armaan457 Participating under SSOC'24 I have the working knowledge of Neural Networks and CNNs and would really like to work on this. I'll be implementing this using TensorFlow trying different activation functions and layers. Can I be assigned this? Thank you!
> 
> Can you implement 4-5 models for this dataset/project?

I am a beginner (1st year UG student) but I'll try my best to do so with the highest accuracy I can achieve.

**Comment by fspzar123** [comment]
Hey,
Can You Please Assign this issue under SSOC 2024 Season 3
Filbert Shawn
Github:- https://github.com/fspzar123
I am pretty good with Neural Networks and CNN
Thanks.
Participation Role:- SSOC Season 3

**Comment by abhisheks008** [comment]
> > > Armaan Jagirdar Github: https://github.com/Armaan457 Participating under SSOC'24 I have the working knowledge of Neural Networks and CNNs and would really like to work on this. I'll be implementing this using TensorFlow trying different activation functions and layers. Can I be assigned this? Thank you!
> > 
> > 
> > Can you implement 4-5 models for this dataset/project?
> 
> Yes sir, I am a beginner (1st year UG student) but I'll try my best to do so with the highest accuracy I can achieve.

Assigned to you @Armaan457
